README.md

```diff
@@ -76,46 +81,82 @@ For example to run all unit tests execute in the build directory:
 ```
 ctest -L unit
 ```
+### Build Instructions for Windows using Visual Studio
+
+First, install Visual Studio and select the "Desktop development with C++" workload. 
+Ensure that CMake and wget are installed and added to your PATH.
+
+Open a command prompt and run:
+```
+cd path\to\marabou\repo\folder
+mkdir build 
+cd build
+cmake .. -G"Visual Studio 15 2017 Win64"
+cmake --build . --config Release
+```
+This process builds Marabou using the generator "Visual Studio 15 2017 Win64". 
+For 32-bit machines, omit Win64. Other generators can also be used, 
+but "Visual Studio 15 2017 Win64" has been tested.
 
+The Marabou executable file will be written to the build/Release folder. To build in 
+Debug mode, simply run "cmake --build . --config Debug", and the executables will be 
+written to build/Debug.
 
 ### Python API
 Using Marabou through the Python interface requires Python 3. It may be useful
 to set up a Python virtual environment, see
 [here](https://docs.python.org/3/tutorial/venv.html) for more information.
 
-The Python interface requires *pybind11* (which is automatically downloaded), to
-build also the python API run
+The Python interface requires *pybind11* (which is automatically downloaded). 
+To also build the python API on Linux or MacOS, run:
 ```
 cd path/to/marabou/repo/folder
 mkdir build 
 cd build
 cmake .. -DBUILD_PYTHON=ON
 cmake --build .
 ```
+On Windows, run:
+```
+cd path\to\marabou\repo\folder
+mkdir build 
+cd build
+cmake .. -G"Visual Studio 15 2017 Win64" -DBUILD_PYTHON=ON
+cmake --build . --config Release
+```
 This process will produce the binary file and the shared library for the Python 
-API, the shared library will be at maraboupy folder
-
+API. The shared library will be in the maraboupy folder for Linux and MacOS. 
+On Windows, the shared library is written to a Release subfolder in maraboupy, 
+so you will need to move the Release/\*pyd file to the maraboupy folder:
```

Comment:
We should keep the same behavior, you can easily copy the target with CMake command (talk to me if needed).

But I actually was rethinking about it and not sure we should copy it at all, @guykatzz  @mykelk  what do you think? when building marabou we create the Python API shared library, should we copy it to the maraboupy (and make it easy to use), or keep it in the build folder and keep the build "clean" (i.e. the build does not create any thing outside the dir)

Reply:
Yeah, keeping it clean might be best.

Reply:
I agree with Mykel. We can keep it in the build folder until there's a reason to do otherwise.

Reply:
How would we be able to run maraboupy from python and not move the python dynamic link library file to the maraboupy folder?

Reply:
The correct way to do it is:
install the python library for users (make install, that will put the .so in /usr/lib )
edit the python path for developers (add build/lib/*.so to the PYTHONPATH variable), this is not a permanent solution because need to edit the PYTHONPATH every time opening a new terminal.

I think that for this pull request we can keep the situation as is, and I will open a new issue and will solve it in the near future, sounds good?

Reply:
I see, and currently users can edit .bashrc (or environmental variables on Windows) so that PYTHONPATH is already edited for every new terminal, which is a quick one time change.

I agree that a separate issue and pull request for this makes sense.